### ISSUE TITLE
Parse XML as String [Feature Request]

### ISSUE THREAD

**Issue by maarest** [databricks/spark-xml#119]
Sample XML

```
<rowtag>
   <element>a</element>
   <nested><nest>value</nest></nested>
</rowtag>
```

I'd like to be able to give a schema element:string, nested:string, such as the value of nested is 
`"<nest>value</nest>"`
. Right now if I provide a schema with nested as StringType, it fails with a Scala MatchError

This is an improvement / feature request and not a bug. Don't know how hard it would be to implement?

Cheers!
Stephane

**Comment by HyukjinKwon** [comment]
@maarest Thanks for reporting this and pointing this out precisely. Actually, I found this problem too. This PR, https://github.com/databricks/spark-xml/pull/107, will solve that issue. I will test this again with the PR.

**Comment by maarest** [comment]
any news on the testing?

**Comment by HyukjinKwon** [comment]
@maarest Sorry, I have been busy. I realised I gave a wrong link. This was actually fixed in https://github.com/databricks/spark-xml/pull/106 and included in 0.3.3. So,

with the data below

```scala
<people>
  <rowtag>
    <element>a</element>
    <nested><nest>value</nest></nested>
  </rowtag>
</people>
```

this code below: 

```scala
val schema = StructType(
  Seq(
    StructField("element", StringType, nullable = true),
    StructField("nested", StringType, nullable = true)))
val df = sqlContext.read
  .format("xml")
  .option("rowTag", "rowtag")
  .schema(schema)
  .load(agesFile)
```

will produce the results below:

```bash
root
 |-- element: string (nullable = true)
 |-- nested: string (nullable = true)

+-------+------------------+
|element|            nested|
+-------+------------------+
|      a|<nest>value</nest>|
+-------+------------------+
```
